transform/operations.go

```diff
@@ -357,7 +360,36 @@ func (c *convertToFloat64) transform(raw interface{}) (interface{}, error) {
 	}
 }
 
-// requiredArgs checks the given args map to make sure it contains the required args and only the required args.
+// convertToInt64 is a transformOperation which converts various types to int64.
+type convertToInt64 struct {
+	args map[string]string
+}
+
+func (c *convertToInt64) init(args map[string]string) error {
+	return nil
+}
+
+func (c *convertToInt64) transform(raw interface{}) (interface{}, error) {
+	switch in := raw.(type) {
+	case string:
+		return strconv.ParseInt(in, 2, 64)
+	case int:
+		return int64(in), nil
+	case int8:
+		return int64(in), nil
+	case int16:
+		return int64(in), nil
+	case int32:
+		return int64(in), nil
+	case int64:
+		return int64(in), nil
+	default:
+		return nil, fmt.Errorf("convertToInt64 only supports strings, int, and float64, raw type: %T", raw)
```

Comment:
You don't have `float64` in the list.

Reply:
I was curious about that. Further raises the question of how to handle the decimal points. Should there be rounding, or should the decimals simply be removed? or should it throw an error?

Reply:
When using this kind of function I would expect the intention is for the source to not have a fractional part to begin with.  The caveat is sometimes floating-point can't exactly represent a number so what you think would be an integer isn't due to imprecision, yet int64 is likely big enough to represent it.   So I would just truncate it.

Reply:
https://github.com/GannettDigital/jstransform/pull/123/commits/8bcccf200d586655e58e513a2707b3a9d8a43004